apps/site/assets/ts/stop/components/amenities/AmenityCard.tsx

```diff
@@ -29,16 +29,32 @@ export const AmenityModal = ({
   );
 };
 
+export const AmenityLink = ({
+  url,
+  text
+}: {
+  url: string;
+  text: string;
+}): JSX.Element => (
+  <p>
+    <a href={url} className="c-call-to-action">
+      {text}
+    </a>
+  </p>
+);
+
```

Comment:
Since this pattern is commonly used throughout dotcom I wonder if this should be its own component/named for reuse across the application

Reply:
There are some usages which are wrapped in a `<div>` or contain additional attributes on the `<a>`, which makes developing a reusable version a bit more complex, but seems doable. Though a lot of the usages of this pattern are not in React...